src/cpp/xmlparser/XMLParser.cpp

```diff
@@ -102,6 +114,10 @@ XMLP_ret XMLParser::parseXML(tinyxml2::XMLDocument& xmlDoc, up_base_node_t& root
                 {
                     ret = parseXMLTopicData(node, *root);
                 }
+                else if (strcmp(tag, LOG) == 0)
+                {
+                    ret = parseLogConfig(node);
+                }
```

Comment:
Can you add an `else`? It has to print and return an error.

Reply:
Added